### PR TITLE
Adding V0.1 Mods 

### PR DESCRIPTION
**Pull request by JJShankles**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
I created a dual color skirt mod for the Voron V0.1. I added angles to the Skirt parts so that with a filament swap you can get a dual color effect. It is similar to an existing mod for the V0, but with the longer legs on the V0.1 the old mod doesn't fit anymore.

I modified another existing mod to add an exhaust fan on the Voron V0.1. The original mod replaced one of the tophat panels, but I made it cover the belts on the back of the printer. This mounts using the holes that would normally hold on the back electronics cover. This exhaust uses a roomba HEPA filter and activated carbon sheets as a filter device.

#### How was it tested? 
I have both of the mods running currently on my printer, and they are working great.

#### Screenshots (if appropriate)
I added several pictures in the commits.

### CODE REVIEW

File: printer_mods/JJShankles/Motor_Cover_HEPA_Exhaust_Filter/Images/test [hunk 1]
```diff
@@ -0,0 +1 @@
+test file
```
Comment (cbc02009) [inline printer_mods/JJShankles/Motor_Cover_HEPA_Exhaust_Filter/Images/test:1]: remove the empty file

File: printer_mods/JJShankles/V0.1_Dual_Color_Hex_Skirt/Images/test [hunk 2]
```diff
@@ -0,0 +1 @@
+new file to create files
```
Comment (cbc02009) [inline printer_mods/JJShankles/V0.1_Dual_Color_Hex_Skirt/Images/test:1]: remove the empty file

File: printer_mods/README.md [hunk 3]
```diff
@@ -173,6 +173,8 @@ like so:
 |jasonc |[V2.4 Z-Idler Shoulder Bolt Mod](./jasonc/Z_Idler_Shoulder_Bolt) |Modification to the idler carriers to accept a 25mm shoulder bolt. Ideal for Gates idlers. |:x: |:x: |:heavy_check_mark: |:x:|
 | Jasrags | [Feed slot with filament pass-through](./Jasrags/sw_fee_slot_filament_passthrough) | A feed slot with a filament pass-through | :x: | :x: | :x: | :heavy_check_mark: |:x:|
 |JD |[V2.4 Filament runout sensor](./JD/RefillPlease) | Replace the 2.4 stock bowden tube holder and add a filament runout sensor  | :x: | :heavy_check_mark: | :heavy_check_mark: | :grey_question: | :heavy_check_mark: |
+| JJShankles | [Motor Cover HEPA Exhaust Filter](./JJShankles/Motor_Cover_HEPA_Exhaust_Filter) | HEPA and Carbon Filter that covers motors at back of printer | :heavy_check_mark: |:x: |:x: |:x: |:x: |
```
Comment (cbc02009) [inline printer_mods/README.md:176]: your entry should go below jgehrig

**Review by cbc02009 — CHANGES_REQUESTED** [review]
Please remove all spaces from file names. Please add a CAD folder to each mod including the STEP file source for each part.

**Comment by cbc02009** [comment]
Still need to look at stl orientation and links.

**Comment by MapleLeafMakers** [comment]
@JJShankles If you still want these included in the repo, please consider closing this and opening a separate pull request for each mod.  It's very difficult for us as volunteers with limited time to review multiple mods in a single pull request.

**Comment by cbc02009** [comment]
Closing due to inactivity.

**Pull request closed without merging**